sql-plugin/src/main/scala/org/apache/spark/sql/rapids/stringFunctions.scala

```diff
@@ -695,17 +695,7 @@ case class GpuStringRepeat(input: Expression, repeatTimes: Expression)
   }
```

Comment:
2023 copyrights

Reply:
Oh wow.

Reply:
I'm taking this into draft, waiting for its dependency.